tests/chainer_tests/functions_tests/connection_tests/test_embed_id.py

```diff
@@ -13,7 +13,9 @@
 
 @testing.parameterize(*testing.product_dict(
     [
-        {'x': (0, 1, 2), 'w_shape': (5, 3), 'ignore_label': 0},
+        {'x': (0, 1, 2), 'w_shape': (5, 3), 'ignore_label': None},
```

Comment:
Lets try to find the reason of the failure instead of hacking our way out.

Reply:
I think this is because the backward pass uses `broadcast_arrays` function which is not supported in ChainerX. I think when we use the ChainerX implementation of `embed_id` (in `forward_chainerx`), this error will not come.

Reply:
I will double check it.
It should use a fallback in such case ...

Reply:
What changes should I make for this?

Reply:
I will look into it later today, sorry

Reply:
I added a fallback for `broadcast_arrays` in #7960 
please revert this change and lets test it after that.

Reply:
sure!

Reply:
the ignore_label =-1 is still missing